roles/pulp_webserver/tasks/apache.yml

```
@@ -12,19 +12,35 @@
         dest: "{{ pulp_webserver_apache_vhost_dir }}/pulp-vhost.conf"
       notify: reload apache
 
-    - name: Enable Apache vhost files and mod files
+    - name: Enable Apache mod files
+      apache2_module:
+        state: present
+        name: "{{ item }}"
+      loop:
+        - headers
+        - proxy
+        - proxy_http
+        - ssl
+      when: ansible_facts.os_family == "Debian"
+      notify: reload apache
+
+    - name: Enable Apache vhost files
       file:
         src: "{{ item.src }}"
         dest: "{{ item.dest }}"
         state: link
       loop:
         - src: ../sites-available/pulp-vhost.conf
           dest: /etc/apache2/sites-enabled/pulp-vhost.conf
-        - src: ../mods-available/proxy.load
-          dest: /etc/apache2/mods-enabled/proxy.load
       when: ansible_facts.os_family == "Debian"
       notify: reload apache
 
+    - name: Remove default apache vhost file
+      file:
+        state: absent
+        path: /etc/apache2/sites-enabled/000-default.conf
+      notify: reload apache
```

```suggestion
      when: ansible_facts.os_family == "Debian"
      notify: reload apache
```
The /etc/apache2 folder is only used on the Debian family. The alternative is to add a comment.